packages/fxa-settings/src/components/UnitRowRecoveryKey/index.tsx

```diff
@@ -75,17 +75,20 @@ export const UnitRowRecoveryKey = () => {
       </LinkExternal>
       {modalRevealed && (
         <VerifiedSessionGuard onDismiss={hideModal} onError={onError}>
-          {/* TODO customize confirm button text and style */}
           <Modal
             onDismiss={hideModal}
             onConfirm={deleteRecoveryKey}
+            // TODO - Uncomment these two
+            // lines once #6302 is merged
```

Comment:
Let's comment in that PR about this line so that can be done there as I think this will be merged first